source/reference/connection-string.txt

```diff
@@ -51,6 +51,8 @@ The components of this string are:
        specific database using these credentials after connecting to
        the :program:`mongod` instance.
 
+       If username or password contains an at-sign ("@") or colon (":") then the username and/or password must be URL encoded. At-sign ("@") is replaced with "%40" (3 characters), while colon (":") is replaced with "%3A" (3 characters).
+
```

Comment:
Sorry for the noise, but it does seem to be more than just the "@" and ":"
I'll go with my first suggestion then.